app/controllers/concerns/feed.rb

```diff
@@ -1,17 +1,10 @@
 module Feed
   def respond_for_feed(events)
     if request.xhr?
-      render_xhr_response_for_feed(events)
-    else
-      render 'shared/feed'
-    end
-  end
-
-  def render_xhr_response_for_feed(events)
-    if events.any?
+      response.headers['TotalPages'] = events.total_pages
```

Comment:
I use this method both in users feed and common feed. I put it in concern for DRY